### PR TITLE
SOV-3847: Lend and borrow section

### PR DESCRIPTION
**Pull request by rick23p**
https://sovryn.atlassian.net/browse/SOV-3847

### CODE REVIEW

File: apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.constants.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+export const COLUMNS_CONFIG = [
+  {
+    id: 'asset',
+    title: t(translation.asset),
+    cellRenderer: (pool: LendingPool) => (
+      <div className="flex items-center font-medium text-base sm:text-xs">
+        <AssetRenderer
+          showAssetLogo
+          asset={pool.getAsset()}
+          className="lg:justify-start justify-end"
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.constants.tsx:27]:
```suggestion
          className="lg:justify-start justify-end"
          assetClassName="text-xs font-medium"
```

File: apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.constants.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { NextSupplyInterestRate } from '../../../LendPage/components/NextSupplyInterestRate/NextSupplyInterestRate';
+
+const translation = translations.protocolDataPage.lendAndBorrow;
+
+export const COLUMNS_CONFIG = [
+  {
+    id: 'asset',
+    title: t(translation.asset),
+    cellRenderer: (pool: LendingPool) => (
+      <div className="flex items-center font-medium text-base sm:text-xs">
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.constants.tsx]:
```suggestion
      <div className="flex items-center">
```

File: apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx [hunk 3]
```diff
@@ -1,19 +1,111 @@
...
+
+export const LendAndBorrow: FC = () => {
+  const navigate = useNavigate();
+
+  const generateRowTitle = useCallback(
+    (pool: LendingPool) => (
+      <AssetRenderer
+        showAssetLogo
+        asset={pool.getAsset()}
+        className="lg:justify-start justify-end my-2"
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx]:
```suggestion
        className="lg:justify-start justify-end my-2"
        assetClassName="text-base font-medium"
```

File: apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx [hunk 4]
```diff
@@ -1,19 +1,111 @@
...
-      children={t(pageTranslations.sections.lendAndBorrow)}
-    />
-    <div className="min-h-72 mt-5">
-      <p>TODO: Add the actual content here</p>
+const lendPools = LendingPoolDictionary.list();
+
+export const LendAndBorrow: FC = () => {
+  const navigate = useNavigate();
+
+  const generateRowTitle = useCallback(
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx:30]: on mobile row missed `Borrow APR` info, you can check how its implemented in `LendFrame`, its pretty the same

**Review by pietro-maximoff — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx (hunk 4 above)
Comment (rick23p) [reply apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx:30]: Well on the design it's not showing the APR. and if you think about it it's not clear to show borrow or lend APR in title.
That's why i'm showing both values on the content

Comment (pietro-maximoff) [reply apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx:30]: @rick23p it's doesn't show `Borrow APR` when the row is open, but when it's closed it does.
borrow

**Review by pietro-maximoff — APPROVED** [review]

**Pull request merged**